### ISSUE TITLE
code snippets' ContentManager should be configurable

### ISSUE THREAD

**Issue by icankeep** [jupytercalpoly/jupyterlab-code-snippets#207]
## Detail
We need to configure a persistent directory because all of our labs are running on docker. And I think it would be better that code snippets' ContentManager is configurable.

**Comment by jahn96** [comment]
what version of this extension are you using?

**Comment by jahn96** [comment]
Are you having an issue similar to this?
https://github.com/jupytercalpoly/jupyterlab-code-snippets/issues/115
Otherwise, could you elaborate the issue you are experiencing?

**Comment by icankeep** [comment]
@jahn96 Hello, our service is based on jupyterhub. Each user has his own docker container, and each container has a labhub process. We really want to use this plugin, but the code snippets generated by this plugin will disappear after docker is destroyed, so We hope that this directory can be configured as our custom remote persistent directory. 

> Are you having an issue similar to this? #115
Before this pr, the code snippets are saved in the `snippets` directory. After this pr, the code snippets seems to be saved in config file, it can not solve my problem. All files in docker will destroy. 
And Stored in the configuration file, I am worried that if the code snippet is too large, will it affect the stability of the service?

**Comment by jahn96** [comment]
I agree with you that if the code snippet is too large, it can affect the stability of the service. We will put this issue on high priority and work to figure something out in the next cycle. Thanks!
